Unaccountable-Magpie/BangazonSite
# Got a ticket done!

### Purpose:
getting the users and product types to show up in dropdowns
also got the column titles in product type details

### How it fits in context of the project:
(Link to ticket: # )
#5  and #53 

### Specific Feature affected:
product type details and product create


### How to test (Be thorough!):

1. git fetch --all

git checkout jr-stuff

open nugget console and select bangazonTests and run dotnet test. You should get 4 passes and 0 fails

after that push execute

in the web page login and click "sell something" and you should see options in the dropdowns

after that, click "product types" and then click on "clothing" and you should see the column titles i added. ex: Price, Quantity
